.github/workflows/MacOS.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         mkdir build 
         cd build
         cmake -DTEST_FILE_DIR=/Users/runner/data -DCMAKE_INSTALL_PREFIX=./install -DENABLE_PYTHON=ON ..
-        make -j2 VERBOSE=1
+        make VERBOSE=1
         make install
         cd python
```

Comment:
I did see you #314, but for consistency should we also remove the `-j2` from the other yml scripts as well?  Or were you waiting to see if that resolved the problem on MacOS before making this change across the board?

Reply:
I intend to put the j2 back in, as soon as the rest of the tests are converted.`